lib/docx/body-reader.js

```diff
@@ -312,8 +312,10 @@ function BodyReader(options) {
             
             var gridSpan = properties.firstOrEmpty("w:gridSpan").attributes["w:val"];
             var colSpan = gridSpan ? parseInt(gridSpan, 10) : 1;
+            var isTableHeader = properties.children && properties.children[0].attributes['w:firstColumn'] === '1';
```

Comment:
Again, reading whatever the first child is feels dangerous, and if this is a boolean property, then the code to read boolean properties should be used so that all cases are covered.